arcade/__init__.py

```diff
@@ -9,7 +9,7 @@
 # Error out if we import Arcade with an incompatible version of Python.
 import sys
 import os
-from datetime import datetime
+from datetime import datetime, _TzInfo
```

Comment:
My initial PR might have been a little busted due to my local nodejs breaking my pyright (I know pyright shouldn't depend on node, but it does :cry:). This should work (I think):
```suggestion
from datetime import datetime, tzinfo
```
The Python source for the datetime module shows it as a real ABC, and it should be less brittle than `_TZInfo`.